source/features/editor-link-new-tab.tsx

```diff
@@ -0,0 +1,29 @@
+import * as pageDetect from 'github-url-detection';
+import delegate, {DelegateEvent} from 'delegate-it';
+
+import features from '../feature-manager.js';
+
+function openInNewTab(event: DelegateEvent<MouseEvent, HTMLLinkElement>): void {
+	event.preventDefault();
+	window.open(event.delegateTarget.href, '_blank');
+}
+
+function init(signal: AbortSignal): void {
+	delegate('div.js-preview-body a, div.html-blob a', 'click', openInNewTab, {signal});
```

Comment:
```suggestion
	delegate([
'div.js-preview-body a', // on page abc
'div.html-blob a', // on page def
], 'click', openInNewTab, {signal});
```

We should describe each selector wherever possible